scripts/hf-skippy-mtp-certify-job.py

```
@@ -0,0 +1,168 @@
+
+def projector_path(args: argparse.Namespace) -> Path:
+    if not args.projector_url:
+        return require_gguf_magic(Path(args.projector))
+    scheme = urllib.parse.urlparse(args.projector_url).scheme
+    if scheme not in ("http", "https"):
+        raise RuntimeError(f"unsupported projector URL scheme: {scheme!r}")
+    target = Path(args.projector_local_path)
+    target.parent.mkdir(parents=True, exist_ok=True)
+    temporary = target.with_suffix(f"{target.suffix}.part")
+    print(f"+ download {args.projector_url} -> {target}", flush=True)
+    with urllib.request.urlopen(args.projector_url) as response, temporary.open("wb") as output:
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)